internal/statemachine/classic_test.go

```diff
@@ -15,95 +14,6 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
-var testCaseName string
```

Comment:
These are very useful, and I needed them for the newly added helper_test.go file. I moved them out of class_test.go since they are not going to be used just for classic tests anymore.